ClientApp/src/app/flights/flights.component.html

```diff
@@ -0,0 +1 @@
+<p>flights works!</p>
```

Comment:
Please do not include non-functional and unfinished components in pull requests.